source/form.html

```diff
@@ -0,0 +1,187 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+<head>
+  <meta charset="UTF-8">
+  <title>Cat Energy</title>
+</head>
+<body class="page__body">
+  <header class="main-header">
+    <a class="logo" href="#">
+      <img class="logo-image" src="img/logo.svg" width="202" height="59"
+      alt="Логотип «Кэт Энерджи»">
+    </a>
+    <nav class="main-nav">
+      <button class="main-nav__toggle" type="button"><span class="visually-hidden">Открыть меню</span></button>
+      <div class="main-nav__wrapper">
+        <ul class="main-nav__list site-list">
+          <li class="site-list__item site-list__item--current">
+            <a>
+              Главная
+            </a>
+          </li>
+          <li class="site-list__item">
+            <a href="#">
+              Каталог продукции
+            </a>
+          </li>
+          <li class="site-list__item">
+            <a href="#">
+              Подбор программы
+            </a>
+          </li>
+        </ul>
+    </nav>
+  </header>
+
+  <main class="main-container">
+    <section class="form">
+      <div class="form__wrapper">
+        <h1 class="form__title">Подбор программы</h1>
+        <p class="form__description">Заполните анкету, и мы подберем программу питания для вашего кота</p>
+        <div class="form__profile">
+          <ul class="form__list--profile">
```

Comment:
Модификатор без основного класс. Ниже у аналогичных блоков так же.
```suggestion
          <ul class="form__list form__list--profile">
```